Dockerfile

```
@@ -9,8 +9,8 @@ RUN mkdir /firefly/frontend \
 ADD . .
 RUN make build
 
-FROM golang:1.16-alpine3.13 AS fabric-builder
-RUN apk add  libc6-compat
+FROM golang@sha256:bab81aadc644eee23bafdf479e9bd60c418d405dbc79370ababbb2a2f88d8034 AS fabric-builder
```

I'm really worried about how deep we're burying dependency information that needs to be regularly updated in the Dockerfiles at the moment.

Specifically:
- UI release version
- This pretty obscure tag that links to a particular (I assume ARM friendly) image of Golang

Can these be rolled up to the Manifest?

---

It's a very good point. I think it's possible to pull these into the manifest if we are okay with moving beyond just a basic `docker build ...` command. Perhaps we can add a target to the Makefile for `docker` so the command just becomes `make docker`. And that script takes care of passing the versions from the manifest to here or wherever else they need to go.

---

As an aside, this is just the sha for the x86_64 version of this image (which runs just fine on an M1 Mac). It is the same image that this Dockerfile would have picked if it were running on an x86_64 host.

---

Think `make docker` sounds like a good approach.

Other option which would move forwards with this PR as is, is to put in the comments a repeatable instruction to find (without owning an M1 Mac) the reverse lookup to "what Go image is this?" and most importantly the next x86_64 image to apply a Go patch release (common security update requirement).

---

```suggestion
FROM --platform=linux/x86_64 golang:1.16-alpine3.13 AS fabric-builder
```

This seemed to work for this step for me @nguyer - I think it's doing the same thing as your change